python/pyspark/sql/readwriter.py

```diff
@@ -1259,12 +1218,15 @@ def parquet(self, path, mode=None, partitionBy=None, compression=None):
                 exists.
         partitionBy : str or list, optional
             names of partitioning columns
-        compression : str, optional
-            compression codec to use when saving to file. This can be one of the
-            known case-insensitive shorten names (none, uncompressed, snappy, gzip,
-            lzo, brotli, lz4, and zstd). This will override
-            ``spark.sql.parquet.compression.codec``. If None is set, it uses the
-            value specified in ``spark.sql.parquet.compression.codec``.
+
+        Other Parameters
+        ----------------
+        Extra options
+            For the extra options, refer to
+            `Data Source Option <https://spark.apache.org/docs/latest/sql-data-sources-parquet.html#data-source-option>`_  # noqa
+            and
+            `Generic File Source Options <https://spark.apache.org/docs/latest/sql-data-sources-generic-options.html`>_  # noqa
```

Comment:
Here you linked generic options but here https://github.com/apache/spark/pull/32204/files#diff-769505539e448f769d060095f72f4687c10b445d52895dbe3ef3668dea7b7cbeR461 you didn't. What's difference?

Reply:
Oh,, It was my mistake. I updated, thanks!